keyboards/hotdox/config.h

```diff
@@ -3,13 +3,14 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0x414B // 'AK' Alpaca Keyboards
+
+#define VENDOR_ID       0x414B // 'AK' for Alpaca Keyboards
 #define PRODUCT_ID      0x0001 // sequentially numbered
 
 #define DEVICE_VER      0x0001
-#define MANUFACTURER    Z.P.Z.O.
-#define PRODUCT         Alpaca Keyboards HotDox
-#define DESCRIPTION     QMK keyboard firmware for HotDox
+#define MANUFACTURER    Alpaca Keyboards
+#define PRODUCT         ErgoDox 76 "HotDox"
```

Comment:
@fauxpark using quotes here okay?

Reply:
Oh, should I use single quotes here instead? The name of the board has quotes like that and I thought those should be preserved. Do double quotes interfere with the markup?

Reply:
Quotes are fine here. The main problem with quotes is when they are around the entire string, you might expect them to be stripped so that the product name is what's inside the quotes, but that is not the case due to how the stringizing works.